Adding 6 headshots for Q4 community spotlight

amada-ech opened the pull request:
For the Q4 community spotlight (Nov 2023 -Jan 2024)

## What are you changing in this pull request and why?
<!---
I am uploading 6 headshots for community members we will feature in the Community spotlight for Q4 of year 2023. They are the Community Award winners.
-->

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

amada-ech committed:
Adding 6 headshots for Q4 community spotlight

For the Q4 community spotlight (Nov 2023 -Jan 2024)

amada-ech committed:
Create dakota-kelley.md

mirnawong1 commented on website/docs/community/spotlight/dakota-kelley.md:
```diff
@@ -0,0 +1,32 @@
+---
+id: #dakota-kelley
```
i think this doesn't require the hash, going to remove it for you

```suggestion
id: dakota-kelley
```

mirnawong1 committed:
Update website/docs/community/spotlight/dakota-kelley.md

mirnawong1 committed:
Merge branch 'current' into amada-ech-patch-1

mirnawong1 committed:
Merge branch 'current' into amada-ech-patch-1

amada-ech committed:
Update fabiyi-opeyemi.md

updating dateCreated so that Opeyemi transfers from Q3 spotlight to Q4 (Coalesce) award recipients)

amada-ech committed:
Update bruno-de-lima.md

updating bruno's answers to:
- About section 
-When did you join the dbt community and in what way has it impacted your career? "

Also - I updated the date so he shows up for Q4

amada-ech committed:
Update karen-hsieh.md

-title change from Director of Product & Data to "Director of Tech & Data"
-date change from  2023-03-24 to 2023-11-03

amada-ech committed:
Update josh-devlin.md

Change date from 2023-06-27 to 2023-11-04

edit About Me section

amada-ech committed:
Update bruno-de-lima.md

removed a / after an https link

amada-ech committed:
Update dakota-kelley.md

deleting this since he doesn't have twitter

  - name: Twitter
    link: link

amada-ech committed:
Update bruno-de-lima.md

fixed a link on line 9

amada-ech committed:
Update bruno-de-lima.md

Added tabs so that everything under description: needs to be consistently tabbed over.

amada-ech committed:
Update dakota-kelley.md

Added tabs so that everything under description: needs to be consistently tabbed over.

amada-ech committed:
Update fabiyi-opeyemi.md

Added tabs bc everything under description: needs to be consistently tabbed over.

amada-ech committed:
Create sydney-burns.md

Adding Sydney Burns to Spotlight Q4

amada-ech committed:
Create oliver-cramer.md

Adding Oliver

amada-ech committed:
Create sam-debruyn.md

sam's page

amada-ech committed:
Update oliver-cramer.md

adding oliver's photo, i had forgotten to edit the "image" line 6

amada-ech committed:
Update dakota-kelley.md

removed excess "tabs" in description

amada-ech committed:
Create stacy-lo.md

stacy's entry

amada-ech committed:
Create alison-stanton.md

alison's page

amada-ech committed:
Update stacy-lo.md

fixing link to Bruno's name so it opens in new tab

amada-ech committed:
Update index.js

updating spotlight date

JKarlavige committed:
Update bruno-de-lima.md

fixes meetup link

JKarlavige commented on website/docs/community/spotlight/dakota-kelley.md:
```diff
@@ -0,0 +1,30 @@
+socialLinks:
+  - name: LinkedIn
+    link: https://www.linkedin.com/in/dakota-kelley/
+dateCreated: 2023-10-03
```
Should this date for Dakota be `2023-11-03`? Currently set to october so will appear under the 'Previous' header.

JKarlavige committed:
Update fabiyi-opeyemi.md

fix coalesce link

JKarlavige committed:
update dakota date

JKarlavige committed:
move spotlight files to correct directory

JKarlavige committed:
adjust links to docs site to be relative

JKarlavige reviewed (approved):
lgtm!

JKarlavige committed:
Merge branch 'current' into amada-ech-patch-1

amada-ech committed:
Update josh-devlin.md

-updating datecreated
-updating community link so that it opens in another tab

amada-ech committed:
Update sydney-burns.md

-edited dateCreated
-added "joined the community" link to Slack

amada-ech committed:
Update dakota-kelley.md

-changed datecreated
-added Slack link to joined the dbt community text
-added link to his Coalesce 2023 talk

amada-ech committed:
Update alison-stanton.md

edited date created
Added hyphens as bullets under "I hope...." to condense spacing

amada-ech committed:
Update fabiyi-opeyemi.md

edited dateCreated

amada-ech committed:
Update bruno-de-lima.md

edited dateCreated
edited Slack link, Discourse Link, and dbt Meetups in Brazil link so they open in new tabs

amada-ech committed:
Update karen-hsieh.md

edit DateCreated

amada-ech committed:
Update sam-debruyn.md

edited date created

changed join the community link to another

hyperlinked github

amada-ech committed:
Update stacy-lo.md

edit DateCreated

amada-ech committed:
Update oliver-cramer.md

edited date created

edited about me and "anything else you want" to add section

nghi-ly committed:
Merge branch 'current' into amada-ech-patch-1
